### DISCUSSION TITLE
Issue getting acces to Student Developer Pack

### DISCUSSION THREAD

**Discussion by KhalSCI** [community/community#111244] [GitHub Education] [answered]
### Hi everyone,

Hey! Am looking for help regarding getting acces to the global campus. My first application got rejected  due to not having 2FA enabled but the 2nd application got approved it was 24 hours but i still did not get acces to any of the benefits. Is this normal? What can i do to acces them. I would love to test the github copilot and many more things!
![gitstudent](https://github.com/community/community/assets/56626157/82c3b021-6aec-4205-b7dd-eb6d67dd0919)

**Comment by amankr1469** [comment, marked as answer]
Ig it takes time to access any of the benefits. I got access after a few days.
